react/week3/react1-fetchapi-week3/src/components/TodoList.jsx

```diff
@@ -0,0 +1,183 @@
+// @ts-nocheck
```

Comment:
A general comment for this component is that i would not put every single component inside one big file. It makes it hard to understand what is going on and to read the component.